Mammoth/TSE/CShip.cpp

```diff
@@ -5356,13 +5356,35 @@ void CShip::OnPaintMap (CMapViewportCtx &Ctx, CG32bitImage &Dest, int x, int y)
 	else if (m_fKnown && m_fShowMapLabel)
 		{
 		CG32bitPixel rgbColor;
-		if (IsEnemy(GetUniverse().GetPOV()))
-			rgbColor = CG32bitPixel(255, 0, 0);
-		else
-			rgbColor = CG32bitPixel(0, 192, 0);
+		rgbColor = GetSymbolColor();
 
-		Dest.DrawDot(x+1, y+1, 0, markerSmallSquare);
-		Dest.DrawDot(x, y, rgbColor, markerSmallFilledSquare);
+		CSovereign* pPlayer = GetUniverse().GetPlayerSovereign();
```

Comment:
We don't have to do it in this PR, but I'd like to consider having the shapes be customizable too.

Reply:
getting them to be customizable will probably require a lot of discussion and thinking about the right way to go about it because of the likelihood of being 'uselessly flexible' to the point of being confusing for the end user, or 'uselessly inflexible' to the point of 'why even bother going to the effort', so we'll make a note of it on ministry and leave that to be tackled in a future ticket.